Convert calculate-agricultural-holiday-entitlement to use ERB templates for questions

floehopper opened the pull request.

chrisroos commented:
Aside from my comment about the indentation in one of the `content_for :body` blocks, this looks good to me.

floehopper commented:
Since #2116 has been merged, I've rebased this against master, re-run the new version of the script, updated the checksums, and force-pushed. You can see the body with fixed indentation in [this commit](https://github.com/alphagov/smart-answers/commit/26e1e4148245e5e1c06f9e574eb84c9cf90a79f1#diff-a7cfc7efbb476ae41d9667299388da21).
